app/controllers/admin/books_controller.rb

```diff
@@ -11,6 +11,7 @@ def new
   def create
     @book = Book.new book_params
     @book.publish_year = DateTime.new(params[:book]["publish_year(1i)"].to_i)
+    @book.image.attach params.dig(:book, :image)
```

Comment:
chị nghĩ em cần check `params.dig(:book, :image)` có khác nil hay kko thì mới attach chứ

```suggestion
    @book.image.attach params.dig(:book, :image) if params.dig(:book, :image).present?
```